### ISSUE TITLE
TalkBack touch exploration in bad state after webview contents are changed

### ISSUE THREAD

**Issue by amirh** [flutter/flutter#30010]
Repro steps:

 * Start the webview example app on Android with TalkBack on.
 * Open the "Navigation Delegate example" from the meatballs menu (this is not a necessary steps, but it's probably easier to debug with a simpler page loaded)
 * Invoke the "refresh" button.
 * Try to use touch exploration to select one of the links in the page(it doesn't work).

Note that after changing the currently focused node (swiping left/right or using touch exploration to select one of the Flutter buttons) touch exploration for the webview works again.
